docs/python_docs/python/tutorials/getting-started/gluon_migration_guide.md

```diff
@@ -19,9 +19,9 @@
 # Gluon2.0: Migration Guide
 
 ## Overview
-Since the introduction of the Gluon API in MXNet 1.x, it has superseded commonly used symbolic, module and model APIs for model development. In fact, Gluon was the first in the deep learning community to unify the flexibility of imperative programming with the performance benefits of symbolic programming, through just-in-time compilation. 
+Since the introduction of the Gluon API in MXNet 1.x, it has superseded commonly used symbolic, module and model APIs for model development. In fact, Gluon was the first in the deep learning community to unify the flexibility of imperative programming with the performance benefits of symbolic programming, through just-in-time compilation.
 
-In Gluon2.0, we extend the support to MXNet numpy and numpy extension with simplified interface and new functionalities: 
+In Gluon2.0, we extend the support to MXNet numpy and numpy extension with simplified interface and new functionalities:
```

Comment:
Shouldn't numpy be written as NumPy? Also there is inconsistency in ndarray - sometimes it's ndarray and sometimes NDArray

Reply:
Fixed for numpy. NDArray is our legacy NDArray whereas ndarray written in small letters refers to NumPy's ndarray.